soconda.sh

```diff
@@ -114,6 +114,10 @@ if [ -e "${confdir}/required_modules.txt" ]; then
     done < "${confdir}/required_modules.txt"
 fi
 
+if [ ${HOSTNAME} = "tiger3.princeton.edu"]; then
+    MPICC=$(which mpicc)
+fi
+
```

Comment:
This is fine for a stop-gap solution right now.  After we get this release out, we should consider adding an optional file in the config directories (e.g. `environment.sh`) which is sourced after the `required_modules.txt` is processed.  That way we can keep system-specific things out of this top-level script.

Reply:
I will change it as soon as we merge the other pr